apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_info.jsx

```diff
@@ -45,6 +46,16 @@ export default class WorkshopEnrollmentSchoolInfo extends React.Component {
     this.props.onDelete(pendingDeleteId);
   }
 
+  csfCourseExperience(csf_course_experience) {
```

Comment:
I'm curious about our naming convention for method, should it start with a verb, or we consider this as an attribute?

Reply:
I think it should start with a verb. I'll fix that.